cli/src/cli_util.rs

```diff
@@ -1604,7 +1653,11 @@ impl WorkspaceCommandHelper {
     }
 
     #[instrument(skip_all)]
-    fn snapshot_working_copy(&mut self, ui: &Ui) -> Result<(), CommandError> {
+    fn snapshot_working_copy(
+        &mut self,
+        ui: &Ui,
+        update_stale: bool,
```

Comment:
Instead of pass `update_stale` in to this function, `SnapshotWorkingCopyResult` can be an error type to be converted to `CommandError` by caller?